src/nvim/ex_cmds2.c

```diff
@@ -3127,21 +3127,21 @@ void ex_language(exarg_T *eap)
       ++_nl_msg_cat_cntr;
 #endif
       /* Reset $LC_ALL, otherwise it would overrule everything. */
-      vim_setenv((char_u *)"LC_ALL", (char_u *)"");
+      vim_setenv("LC_ALL", "");
 
       if (what != LC_TIME) {
         /* Tell gettext() what to translate to.  It apparently doesn't
          * use the currently effective locale. */
         if (what == LC_ALL) {
-          vim_setenv((char_u *)"LANG", name);
+          vim_setenv("LANG", (char *)name);
 
           /* Clear $LANGUAGE because GNU gettext uses it. */
-          vim_setenv((char_u *)"LANGUAGE", (char_u *)"");
+          vim_setenv("LANGUAGE", "");
         }
         if (what != LC_CTYPE) {
           char_u      *mname;
           mname = name;
-          vim_setenv((char_u *)"LC_MESSAGES", mname);
+          vim_setenv("LC_MESSAGES", (char *)mname);
           set_helplang_default(mname);
```

Comment:
Perhaps this could be converted as well, as it's only called in two places, which I think would allow you to do:

```
char *mname = name;
vim_setenv("LC_MESSAGES", mname);
set_helplang_default(mname);
```